tests/test_routes.py

```diff
@@ -341,3 +374,4 @@ def test_get_customers_by_lastname(self):
         # check the data just to be sure
         for customer in data:
             self.assertEqual(customer["lastname"], lastname_to_use)
+>>>>>>> master
```

Comment:
Last one

Reply:
This is due to a merge conflict being pushed into master